CMS/static/scss/crc.scss

```diff
@@ -288,7 +288,7 @@ a {
   text-decoration: none
 }
 
-a:hover, a:focus {
+a:hover {
   color: #646e9b;
```

Comment:
Nice.
Removing tags from this file and moving them to global or specific pages is the goal 👍